src/pybind/mgr/balancer/module.py

```diff
@@ -348,11 +352,32 @@ def show_status(self) -> Tuple[int, str, str]:
         }
         return (0, json.dumps(s, indent=4, sort_keys=True), '')
 
+    @CLIReadCommand('balancer status detailed')
+    def show_status_detail(self) -> Tuple[int, str, str]:
+        """
+        Show balancer status detailed
+        """
+        pg_movement = self.get_osdmap().dump().get('pg_upmap_items', '')
+
+        s = {
+            'plans': list(self.plans.keys()),
+            'active': self.active,
+            'last_optimize_started': self.last_optimize_started,
+            'last_optimize_duration': self.last_optimize_duration,
+            'optimize_result': self.optimize_result,
+            'no_optimization_needed': self.no_optimization_needed,
+            'mode': self.get_module_option('mode'),
+            'pg_last_optimized': pg_movement,
```

Comment:
Now that I'm looking at it, we don't need this piece, which shows all the pg upmap items. This would be redundant information since users can already check this via `ceph osd dump`.

We just need `added_pg_upmap_items` and `removed_pg_upmap_items`.